src/components/buttons/HdLoaderButton.vue

```diff
@@ -15,6 +15,7 @@
       @click="clicked"
       @keyup.enter.space="clicked"
       @transitionend="runTransitionQueue"
+      :disabled="disabled"
```

Comment:
If you're just forwarding the prop to the button, I would go to `v-bind="$attrs"` and remove it from the props definition. That is a minor and just a tip, as other button attributes would be automatically forwarded.